dojo/tools/dependency_check/parser.py

```diff
@@ -37,6 +39,11 @@ def get_finding_from_vulnerability(self, vulnerability, filename, test):
         else:
             severity = self.get_field_value(vulnerability, 'severity').lower().capitalize()
 
+        if severity in SEVERITY:
+            severity = severity
+        else:
+            severity = "Info"
```

Comment:
shouldn't we log something here to trace the fact that something abnormal happened? level warn i'd say or at least info. 
To me this is a dependency check issue, it's not a nominal scenario. It happens for this single vuln. 
Plus it will make it easier to spot issues next time dependency check adds a level and we loose all those findings severity :)

Reply:
also shouldn't we set an intermediate level by default, like "MEDIUM" ? people tend not to look at Info level which increase the chance of missing things that may actually be important

Reply:
We can show a log about this or maybe add a tag value of this severity with a warning tag?

Reply:
@ptrovatelli You beat to your last point! I was just typing that out. Maybe we could also append some tag to the title that says the severity is inaccurate. Logging is definitely a good idea though.